datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -3186,6 +3196,22 @@ private void configureIngestionSourceResolvers(final RuntimeWiring.Builder build
                         })));
   }
 
+  private void configureExecutionRequestResolvers(final RuntimeWiring.Builder builder) {
+    builder
+        .type(
+            "ExecutionRequest",
+            typeWiring -> typeWiring.dataFetcher(
+                    "source", new LoadableTypeResolver<>(
+                            ingestionSourceType,
+                            (env) -> {
+                              final ExecutionRequest executionRequest = env.getSource();
+                              return executionRequest.getInput().getSource().getIngestionSource();
```

Comment:
i think we might want to check to make sure this is not null like we do in some other cases in this file